core/CronArchive.php

```diff
@@ -938,20 +938,19 @@ private function archiveReportsFor($idSite, $period, $date, $archiveSegments, Ti
         // already processed above for "day"
         if ($period != "day") {
 
-            $periodInProgress = $this->isAlreadyArchivingAnyLowerOrThisPeriod($idSite, $period);
-            if ($periodInProgress) {
-                $this->logger->info("- skipping archiving for period '{period}' because processing the period '{periodcheck}' is already in progress.", array('period' => $period, 'periodcheck' => $periodInProgress));
+            if ($this->isAlreadyArchivingUrl($url, $idSite, $period, $date)) {
```

Comment:
In theory we wouldn't need this check and could rely on the one in `->before()` but figured it might be good to have.